docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -42,42 +42,44 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
 
 ### Parameters
 
+* **grouping_expression**
+
+    Specifies the criteria based on which the rows are grouped together. The grouping of rows is performed based on
+    result values of the grouping expressions. A grouping expression may be a column alias like `GROUP BY a`, a column position like
```

Comment:
`column alias` -> `column name` which is probably more accurate.

Reply:
Done